### ISSUE TITLE
[Accessibility]: Contributors page: No landmark heading

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#35353]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34427

### Problem description

There is no heading on this page, so it is difficult for screen readers to understand where they are.

### Expected behavior

We should add a heading to this page

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
We now have breadcrumbs so this is no longer a significant issue, closing!
